src/app/components/main-page/main-page.component.scss

```diff
@@ -15,6 +15,11 @@ nav {
   grid-template-columns: repeat(3, 1fr);
 }
 
+#menu-button {
+  background-color: transparent;
+  text-align: left;
```

Comment:
I think you can use the following bootstrap class for text-align left class="text-left" and transparent can be applied through bg-transparent

Reply:
will do

Reply:
fixed